tests/importance_tests/test_fanova.py

```diff
@@ -71,3 +71,19 @@ def test_fanova_importance_evaluator_with_target() -> None:
     )
 
     assert param_importance != param_importance_with_target
+
+
+def test_fanova_importance_evaluator_single_distribution() -> None:
+    def _objective(trial: Trial) -> float:
+        x = trial.suggest_float("x", 0, 5)
+        y = trial.suggest_float("y", 1, 1)
+        v0 = 4 * x ** 2 + 4 * y ** 2
+        return v0
+
+    study = create_study()
+    study.optimize(_objective, n_trials=3)
+
+    evaluator = FanovaImportanceEvaluator()
+    importances = evaluator.evaluate(study)
+
+    assert importances["y"] == 0.0
```

Comment:
Leaving this test might be fine but do you think it's better suited in https://github.com/optuna/optuna/blob/master/tests/importance_tests/test_init.py ? That is, in the general importance tests, not just fANOVA (although the change is limited to this evaluator).

Reply:
I was thinking about it, but concluded that such test would require to also check the behavior of `MeanDecreaseImpurityImportanceEvaluator`, which i think is out of scope here. How about i tackle this in followup? If it works I'll just add parametrized tests as you suggested, and if not I'll also fix the other evaluator.

Reply:
That sounds like a much better approach. Let me approve this PR as I think it looks good in that case. 👍 
(I think you can start checking the `MeanDecreaseImpurityImportanceEvaluator` while waiting for a second review, if you like of course)

Reply:
I'm fine either way.

The test you added is obviously reasonable since it's testing the behavior of public API, but at the same time, we've left most of the general behavioral checks to `test_init.py`, given that `get_param_importances` is basically just calling `BaseImportanceEvaluator.evaluate`. With that said, for consistency and simplicity, I slightly lean towards removing.